.github/workflows/prerelease.yml

```diff
@@ -0,0 +1,45 @@
+name: publish-prerelease
+on:
+  workflow_dispatch:
+    inputs:
+      pre-release-tag:
+        description: 'The pre-release tag use in the version'
+        required: false
+        default: 'pre'
+      dist-tag:
+        description: 'The dist-tag use'
+        required: false
+        default: 'next'
+      dependencies-to-update:
+        description: 'csv of dependencies to update with the dist-tag'
+        required: false
+        default: '@adobe/aio-lib-runtime,@adobe/aio-lib-web'
```

Comment:
missing the other @adobe deps: https://github.com/adobe/aio-cli-plugin-app/blob/cb1b5c05d4c9afaa7ee57a1b96c5af26c31db4b1/package.json#L8-L17

Reply:
Added all @adobe deps for this plugin